docs/excel/excel-add-ins-workbooks.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Work with workbooks using the Excel JavaScript API
+description: ''
+ms.date: 11/26/2018
+---
+
+
+# Work with Workbooks using the Excel JavaScript API
+
+This article provides code samples that show how to perform common tasks with workbooks using the Excel JavaScript API. For the complete list of properties and methods that the **Workbook** object supports, see [Workbook Object (JavaScript API for Excel)](https://docs.microsoft.com/javascript/api/excel/excel.workbook). This article also covers workbook-level actions performed through other similarly scoped objects, such as [Application](https://docs.microsoft.com/javascript/api/excel/excel.application).
```

Comment:
+1 to Ricky's suggestion to remove "similarly scoped"

Also -- all docs.microsoft.com links should be relative (not absolute) links.  For example:
- incorrect:  `[Workbook Object (JavaScript API for Excel)](https://docs.microsoft.com/javascript/api/excel/excel.workbook)`
- correct: `[Workbook Object (JavaScript API for Excel)](/javascript/api/excel/excel.workbook)`

(FWIW, it looks like we may need to make this change elsewhere in our doc set too...as part of a separate work item.)